src/python/gem5/utils/multisim/__init__.py

```diff
@@ -0,0 +1 @@
+from .multisim import *
```

Comment:
1. this file needs a copyright statement.
2. It's a bit of a pain, but it would be better to specify what you want to import instead of using `*`